findme-users/src/UserService.py

```diff
@@ -8,16 +8,19 @@ def __init__(self, user_repository):
         self.user_repository = user_repository
 
     def post_user(self, data, user_id):
-        return self.user_repository.post_user_to_db({**data, "user_id": user_id})
+        return (self.user_repository.post_user_to_db({**data, "user_id": user_id})
+                .dict(exclude={"scores"}))
```

Comment:
Could it make sense to create a DTO for the user? Excluding the scores all the time might be a bit confusing at some point

Reply:
I'm currently setting up swagger_ui: https://github.com/uzh-ase-fs24/backend/issues/63. Will continue on that tomorrow. For it to work efficiently, we should use Pydantic as often as possible. Therefore, it makes sense to rework this (I can also do it). You can see below how swagger looks like with nice Pydantic models.
![image](https://github.com/uzh-ase-fs24/backend/assets/75266614/4e79f643-03ce-4a79-acd4-a4f9ad305f5d)